html-editor-icons.html

```diff
@@ -96,51 +96,51 @@
           <path d="M9 11a.945.945 0 0 1-1-1V8a.945.945 0 0 1 1-1 .945.945 0 0 1 1 1v2a1 1 0 0 1-1 1z"/>
```

Comment:
I think superscript and subscript should be mirrored here; we might want an icon change to use something more mirrorable than "3." Maybe "i" or "8"?